nss-day-cohort-52/giffygram-retro-blasters
# Jm favorite star

#### Changes Made
1. Added file `filename` to `directoryname` directory.
1. Modified file `filename` to include `functionalityname` and functionality.
​add functionality to the favorite star. When clicked on it changes from blank to yellow.
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.

You can test from the browser. Click on a star and it should change 


4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
